pyproject.toml

```
@@ -1,11 +1,23 @@
 [build-system]
-requires = [
-    "setuptools>=42",
-    "wheel",
-    "scikit-build>=0.16.7",
-    "cmake>=3.18",
-    "nanobind>=1.1.0",
-    "ninja; platform_system!='Windows'"
+requires = ["scikit-build-core", "nanobind>=1.3.0"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "nanobind-example"
+version = "0.0.1"
+description = "An example minimal project that compiles bindings using nanobind and scikit-build"
+readme = "README.md"
+requires-python = ">=3.8"
+authors = [
+    { name = "Wenzel Jakob", email = "wenzel.jakob@epfl.ch" },
+]
+classifiers = [
+    "License :: BSD",
 ]
 
-build-backend = "setuptools.build_meta"
+[project.urls]
+Homepage = "https://github.com/wjakob/nanobind_example"
+
+[[tool.cibuildwheel.overrides]]
+select = "cp312-*"
+config-settings = {wheel.py-api = "cp312"}
```

```suggestion
config-settings = {"wheel.py-api" = "cp312"}
```

Maybe this is needed? Haven't tried setting a nested option this way.

---

Thanks!